compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/Overscroll.kt

```diff
@@ -119,6 +122,17 @@ interface OverscrollEffect {
      * A [Modifier] that will draw this OverscrollEffect
      */
     val effectModifier: Modifier
+
+    /**
+     * Represents an optional overscroll effect that can be applied when fling inertial movement crosses the overscroll area.
+     *
+     * @param scrollValueConverter an instance of [ScrollValueConverter] class which is used to convert the raw scroll offset
+     * into the overscroll effect offset.
+     * @return a nullable instance of the [FlingIntoOverscrollEffect] class, representing the overscroll effect to be used.
+     * If null, no effect will be applied.
+     */
+    fun flingIntoOverscrollEffect(scrollValueConverter: ScrollValueConverter): FlingIntoOverscrollEffect? =
```

Comment:
New API in common won't be available on Android until upstreaming. 1.5 API already frozen, so we need to be careful about it and discuss such changes with Google